distribution/src/main/packaging/systemd/elasticsearch.service

```diff
@@ -15,10 +15,7 @@ WorkingDirectory=/usr/share/elasticsearch
 User=elasticsearch
 Group=elasticsearch
 
-ExecStart=/usr/share/elasticsearch/bin/elasticsearch \
-                                                -p ${PID_DIR}/elasticsearch.pid \
-                                                --quiet \
-                                                --path.conf ${CONF_DIR}
+ExecStart=/usr/share/elasticsearch/bin/elasticsearch -p ${PID_DIR}/elasticsearch.pid --quiet
```

Comment:
It turns out that:
 - the quotes as implemented in this PR are totally busted, the quotes must wrap an entire argument, and arguments are separated by spaces in the unit file syntax
 - the quotes are completely unnecessary, `systemd` will handle unquoted entries correctly

I pushed 46d75a3552a1062a74d6b710ff20b55fc9cb3a8b